services/brig/test/integration/API/TeamUserSearch.hs

```diff
@@ -72,11 +76,48 @@ testEmptyQuerySorted :: TestConstraints m => Brig -> m ()
 testEmptyQuerySorted brig = do
   (tid, userId -> ownerId, users) <- createPopulatedBindingTeamWithNamesAndHandles brig 4
   refreshIndex brig
-  r <- searchResults <$> executeTeamUserSearch brig tid ownerId ""
+  r <- searchResults <$> executeTeamUserSearch brig tid ownerId (Just "") Nothing Nothing Nothing
   let creationDates = fmap teamContactCreatedAt r
   liftIO $
     assertEqual
       "user ids"
       (sort (fmap userId users <> [ownerId]))
       (sort (fmap teamContactUserId r))
   liftIO $ assertEqual "sorted team contacts" (sortOn Down creationDates) creationDates
+
+testSort :: TestConstraints m => Brig -> m ()
+testSort brig = do
+  (tid, userId -> ownerId, usersImplicitOrder) <- createPopulatedBindingTeamWithNamesAndHandles brig 4
+  users <- liftIO $ shuffleM usersImplicitOrder
```

Comment:
`shuffleM` has no effect: you're never using `users` in the order you establish here, you always run `sort` on it first.  (this means you can also change `package.yaml` back, I think.)

Reply:
There seems to be an implicit ordering for 1) users in cassandra 2) users in Elasticsearch. The shuffle exist to mitigate this effect. Without this the test succeeds even if you don't sort in some cases.

Reply:
- [ ] comment this